### PR TITLE
Ref/token store

### PR DESCRIPTION
**Pull request by joan41868**
**Related issue(s)**:
Not closing any issue.

**Summary of the change**:
Implement required changes on stores.

**External impacts**:
None for now.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/store/tokens/unique/OwnerIdentifier.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package com.hedera.services.store.tokens.unique;/*
```
Comment (failfmi) [inline hedera-node/src/main/java/com/hedera/services/store/tokens/unique/OwnerIdentifier.java:1]: nit: Licence

File: hedera-node/src/main/java/com/hedera/services/utils/invertible_fchashmap/IdentifyableUniqueToken.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+ * ‌
+ * Hedera Services Node
+ * Copyright (C) 2018 - 2020 Hedera Hashgraph, LLC
```
Comment (failfmi) [inline hedera-node/src/main/java/com/hedera/services/utils/invertible_fchashmap/IdentifyableUniqueToken.java:4]: 2021, the whole licence should be beneath the package

File: hedera-node/src/test/java/com/hedera/services/store/tokens/unique/OwnerIdentifierTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (failfmi) [inline hedera-node/src/test/java/com/hedera/services/store/tokens/unique/OwnerIdentifierTest.java:1]: licence should be beneath the package

File: hedera-node/src/main/java/com/hedera/services/utils/invertible_fchashmap/IdentifyableUniqueToken.java (hunk 2 above)
Comment (joan41868) [reply hedera-node/src/main/java/com/hedera/services/utils/invertible_fchashmap/IdentifyableUniqueToken.java:4]: This file is a leftover, which returned after my revert. Will be removed.

**Review by failfmi — APPROVED** [review]

**Pull request merged**